synapse/models/inet.py

```diff
@@ -3863,6 +3863,9 @@ def getModelDefs(self):
 
                         ('period', ('ival', {}), {
                             'doc': 'The time period where the channel was available.'}),
+
+                        ('topic', ('str', {'strip': True}), {
```

Comment:
if we're looking for these to fold together, they should probably be lowered as well?